CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 [//]: # (current developments)
 
+## 2.3.0 (2025-06-16)
+
+### Enhancements
+
+* Bump schema version to `1`. (#282 via #283)
```

Comment:
This might be distracting as the change below effectively superseded it. Should we just remove it?

```suggestion
```